element_array_ephys/ephys_acute.py

```diff
@@ -662,6 +664,74 @@ def yield_unit_waveforms():
             self.Waveform.insert(unit_electrode_waveforms, ignore_extra_fields=True)
 
 
+@schema
+class QualityMetric(dj.Imported):
+    definition = """
+    # Clusters and waveforms metrics
+    -> CuratedClustering    
+    """
+
+    class Cluster(dj.Part):
+        definition = """   
+        # Cluster metrics for a particular unit
+        -> master
+        -> CuratedClustering.Unit
+        ---
+        firing_rate=null: float # (Hz) firing rate for a unit 
+        snr=null: float  # signal-to-noise ratio for a unit
+        presence_ratio=null: float  # fraction of time in which spikes are present
+        isi_violation=null: float   # rate of ISI violation as a fraction of overall rate
+        number_violation=null: int  # total number of ISI violations
+        amplitude_cutoff=null: float  # estimate of miss rate based on amplitude histogram
+        isolation_distance=null: float  # distance to nearest cluster in Mahalanobis space
+        l_ratio=null: float  # 
+        d_prime=null: float  # Classification accuracy based on LDA
+        nn_hit_rate=null: float  # Fraction of neighbors for target cluster that are also in target cluster
+        nn_miss_rate=null: float # Fraction of neighbors outside target cluster that are in target cluster
```

Comment:
I am not sure what the difference is here.

Reply:
More details description and specific implementation of these 2 metrics can be found here:
https://github.com/datajoint-company/ecephys_spike_sorting/blob/84f9e596e8519f1088c39bd6c84c25a3b9758217/ecephys_spike_sorting/modules/quality_metrics/metrics.py#L859

Which based on metrics described in Chung, Magland et al. (2017) Neuron 95: 1381-1394 (https://www.sciencedirect.com/science/article/pii/S0896627317307456)

Reply:
Thank you!